app/assets/stylesheets/osem-navbar.css.scss

```diff
@@ -0,0 +1,36 @@
+@import "breakpoints.css.scss";
+.nav-osem {
+    border: none;
+    @include breakpoint(xs) {
+        .navbar-collapse {
+            background-color: white;
+            border-bottom: 2px solid #efefef;
+        }
+        .navbar-nav .black {
```

Comment:
Instead of introducing a new class, you can use this rule, which simply overrides Bootstrap's default rule:

```css
.navbar-nav > li > a {
  color: #555;
}
```